examples/powertools-examples-core/terraform/infra/lambda.tf

```diff
@@ -0,0 +1,112 @@
+resource "aws_lambda_function" "hello_world_lambda" {
+  runtime           = "java11"
+  filename          = "target/helloworld-lambda.jar"
+  source_code_hash  = filebase64sha256("target/helloworld-lambda.jar")
+  function_name     = "hello_world_lambda"
+
+  handler           = "helloworld.App"
+  description       = "Powertools example, deployed by Terraform"
+  timeout           = 20
+  memory_size       = 512
+  role              = "${aws_iam_role.iam_role_for_lambda.arn}"
+  environment {
+    variables = {
+      POWERTOOLS_LOG_LEVEL = "INFO"
+      POWERTOOLS_LOGGER_SAMPLE_RATE = "0.1"
+      POWERTOOLS_LOGGER_LOG_EVENT = "true"
```

Comment:
remove this line, we actually don't handle this env var...

Reply:
Terraform example is based on the CDK example configuration which include these environment variables. Are you sure?

Reply:
yes, we need to cleanup all the examples (#1444)